apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/util/PeerFormat.java

```diff
@@ -25,13 +25,11 @@
  * @author zhaoyuguang
  */
 
-public enum PeerFormat {
-
-    INSTANCE;
+public class PeerFormat {
 
     private static final String ABBR = "...";
 
-    public String Shorten(String original) {
+    public static String shorten(String original) {
         if (!StringUtil.isEmpty(original) && original.length() > Config.Plugin.PEER_MAX_LENGTH) {
             return original.substring(0, Config.Plugin.PEER_MAX_LENGTH - 3) + ABBR;
```

Comment:
What if users set `Config.Plugin.PEER_MAX_LENGTH` to less than `3` (intentionally or by mistake) 😄

Reply:
Can cause unexpected exceptions, the size of the prompt on the document needs to exceed 3 and the judgment of increasing the length of `PEER_MAX_LENGTH ` is less than 3, less than 3 will be invalid and the default value is 200.

Reply:
![image](https://user-images.githubusercontent.com/10150229/61573477-e9e56800-aae1-11e9-972e-1b017a892af9.png)
I found another way, is this better?